react-list.es6

```diff
@@ -86,7 +86,7 @@ export default class extends Component {
 
   componentWillUnmount() {
     window.removeEventListener('resize', this.updateFrame);
-    this.scrollParent.removeEventListener('scroll', this.updateFrame);
+    this.scrollParent && this.scrollParent.removeEventListener('scroll', this.updateFrame);
```

Comment:
Have you found an instance where this is `null`? That would mean `componentWillUnmount` was called before `componentDidMount`, as `componentDidMount` is where `this.scrollParent` is set.